src/main/java/org/folio/inventory/resources/AbstractInstances.java

```diff
@@ -354,7 +355,7 @@ protected String createQueryForPrecedingSucceedingInstances(List<String> instanc
 
   protected String createQueryForRelatedInstances(List<String> instanceIds) {
     String idList = instanceIds.stream().distinct().collect(Collectors.joining(" or "));
-    return format("query=subInstanceId==(%s)+or+superInstanceId==(%s)", idList, idList);
+    return format("subInstanceId==(%s)+or+superInstanceId==(%s)", idList, idList);
```

Comment:
Why has the query= part of this been removed? Is it because the previous `getMany` method expects a raw HTTP query string?

Reply:
Yes: https://github.com/folio-org/mod-inventory/blob/master/src/main/java/org/folio/inventory/storage/external/CollectionResourceClient.java